src/GraphQL.Tests/Utilities/SchemaBuilderExecutionTests.cs

```diff
@@ -61,6 +61,8 @@ public async Task schema_first_generate_exception_with_normal_stack_trace_for_me
                 stack = stack.Remove(0, 5);
             if (stack.StartsWith("   at "))
                 stack = stack.Remove(0, 6);
+            if (stack.StartsWith("   在 "))
+                stack = stack.Remove(0, 5);
```

Comment:
Did these come from a merge conflict?

Reply:
No, I add this code to ensure the test's success in the zh-hans environment.

Reply:
Oh ok